src/store/index.js

```diff
@@ -1,15 +1,17 @@
-/* global window */
 import { createStore, applyMiddleware, compose } from 'redux';
-import { persistStore, persistCombineReducers } from 'redux-persist';
+import { persistStore, persistCombineReducers, createMigrate } from 'redux-persist';
 import storage from 'redux-persist/es/storage'; // default: localStorage if web, AsyncStorage if react-native
 import thunk from 'redux-thunk';
 import reducers from '../reducers';
+import { migrations } from './migrations';
 
 // Redux Persist config
 const config = {
   key: 'root',
+  version: 3,
```

Comment:
should this match the specific version of the app? or just this different migration version as another versioning?

Reply:
I guess that would require a migration config for each patch version which would be pretty pointless

Reply:
This version is just related to the persist